README.md

```diff
@@ -36,57 +36,58 @@ environment variables, or with a link to a configuration guide.
 | AWS_TLS_INSECURE_SKIP_VERIFY | false                    | This should never be set to true, as it disables SSL certificate verification. Used only for development
 | BIND_ADDR                    | :23900                   | The host and port to bind to
-| ELASTIC_SEARCH_URL	       | "http://localhost:11200" | Http url of the ElasticSearch server
+| ELASTIC_SEARCH_URL           | "<http://localhost:11200>" | Http url of the ElasticSearch server
 | GRACEFUL_SHUTDOWN_TIMEOUT    | 5s                       | The graceful shutdown timeout in seconds (`time.Duration` format)
 | HEALTHCHECK_CRITICAL_TIMEOUT | 90s                      | Time to wait until an unhealthy dependent propagates its state to make this app unhealthy (`time.Duration` format)
 | HEALTHCHECK_INTERVAL         | 30s                      | Time between self-healthchecks (`time.Duration` format)
 | SERVICE_AUTH_TOKEN           | ""                       | The service auth token only gets used by the bulk indexer [Running Bulk Indexer](#running-bulk-indexer)
-| ZEBEDEE_URL                  | http://localhost:8082    | The URL to Zebedee (for authorisation)
+| ZEBEDEE_URL                  | <http://localhost:8082>    | The URL to Zebedee (for authorisation)
 
-### API documentation
+## API documentation
 
 Documentation of the API interface is described using swagger 2.0. This specification can be found [here](./swagger.yaml).
 
-### Go SDK - Client Package
+## Go SDK - Client Package
 
 Applications trying to interact with the API can use the Go SDK package which contains a list of client methods that are maintained to align with
 the API. For futher reading on how to use the client follow this [link](./sdk/README.md).
 
-### Connecting to AWS Elasticsearch cluster (dev only)
+## Connecting to AWS Elasticsearch cluster (dev only)
 
-#### Prerequisites
+### Prerequisites
 
-- You will need an user account for the aws account you are trying to connect to
-- You will need to be given a policy to allow read and write access to the AWS Elasticsearch cluster
+* You will need an user account for the aws account you are trying to connect to
+* You will need to be given a policy to allow read and write access to the AWS Elasticsearch cluster
 
-#### Connecting
+### Connecting
 
 To connect to managed Elasticsearch cluster in AWS, you will want to port forward 11200 to the domain endpoint. Using the dp tool, one can do this like so:
 
-```
+```shell
   dp ssh develop <ip of aws box> -p 9200:<elasticsearch cluster domain endpoint e.g. "<unique identifier>" + "eu-west-1.es.amazonaws.com:443"
 ```
 
 Once connected, run the following make target:
 
-```
+```shell
   make local
 ```
 
 ## Running Bulk Indexer
 
 The Bulk Indexer creates an Elastic Search 7.10 index with the alias "ons". It loads data from zebedee, and the dataset API, into the ES 7.10 index.
 
-#### Locally
+### Locally
 
-###### Prerequisites
+#### Prerequisites 2
```

Comment:
Maybe 'Local prerequisites' or 'Prerequisites for running locally' might be more descriptive.

Reply:
done